pubspec.yaml

```diff
@@ -1,72 +1,39 @@
-name: tflite_flutter_helper
+name: tflite_flutter_processing
 description: Easy, fast processing and manipulation input and output of TensorFlow Lite Models.
-version: 0.3.0
-homepage: https://www.github.com/am15h/tflite_flutter_helper
+version: 0.4.1
+homepage: https://www.github.com/lucasoskorep/tflite_flutter_processing
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.10.0"
+  sdk: '>=2.18.0 <4.0.0'
+  flutter: ">=2.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.1.8
-  quiver: ^3.0.1
-  path_provider: ^2.0.1
-  tflite_flutter: ^0.9.0
-  tuple: ^2.0.0
-  camera: ^0.8.1+3
-  ffi: ^1.0.0
-  image: ^3.0.2
+  meta: ^1.12.0
+  quiver: ^3.2.1
+  path_provider: ^2.1.3
+  tflite_flutter: ^0.10.4
+  tuple: ^2.0.2
+  camera: ^0.11.0+1
+  ffi: ^2.1.2
+  image: ^4.2.0
```

Comment:
If we want to release, it should be multiple packages. One for image processor and other for other stuff